core/src/main/java/com/digitalpebble/stormcrawler/parse/JSoupFilters.java

```diff
@@ -75,11 +75,7 @@ public static JSoupFilters fromConf(Map<String, Object> stormConf) {
         return JSoupFilters.emptyParseFilter;
     }
 
-    /**
-     * loads the filters from a JSON configuration file
-     *
-     * @throws IOException
-     */
+    /** loads the filters from a JSON configuration file */
```

Comment:
This might break end user code (removal of a checked exception), so we need to be clear in the changelog.

Reply:
Why should this break end user code? 
It still throws that `IOException` but i removed the empty comment.